### ISSUE TITLE
Remove twisted/test/time_helpers.py

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5820]
|[<img alt="vperic's avatar" src="https://avatars.githubusercontent.com/u/688281?s=50" width="50" height="50">](https://github.com/vperic)| @vperic reported|
|-|-|
|Trac ID|trac#5820|
|Type|task|
|Created|2012-07-28 13:30:57Z|
|Branch|https://github.com/twisted/twisted/tree/remove-timehelpers-5820|

The above module was deprecated in Twisted 10.0 (ticket [#3719](https://github.com/twisted/twisted/issues/3719)) and has a more full-featured replacement (twisted.internet.task.Clock). It's also somewhat internal code which shouldn't have been used outside of Twisted anyway. It's not a real "maintenance burden", but I don't think there's any reason to keep it either - removing it is one less module to port to Python 3.

Attachments:

* [remove-timehelpers.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/1a3/1a39c2c58839d49b069d63b46b40b5c9a3a2d796/0bfe4c6b20f72ffb7340e7fd16384c227a0c2190.patch) (3682 bytes) - added by vperic on 2012-07-28 13:35:24Z - First version

<details><summary>Searchable metadata</summary>

```
trac-id__5820 5820
type__task task
reporter__vperic vperic
priority__normal normal
milestone__ 
branch__branches_remove_timehelpers_5820 branches/remove-timehelpers-5820
branch_author__vperic vperic
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1343482257000000 1343482257000000
changetime__1344897603000000 1344897603000000
version__None None
owner__vperic vperic
cc__thijs
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="vperic's avatar" src="https://avatars.githubusercontent.com/u/688281?s=50" width="50" height="50">](https://github.com/vperic)<a name="note_1"></a>|@vperic commented|
|-|-|

This was simple enough, patch attached.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_2"></a>|@thijstriemstra set owner to @vperic|
|-|-|

"Do not use this module.  It is a lie." Ha :)

Can you apply this patch in a branch and use `force-builds.py` to check if all is good? If the result looks good, please merge. Thanks.

**Comment by twisted-trac** [comment]
|[<img alt="vperic's avatar" src="https://avatars.githubusercontent.com/u/688281?s=50" width="50" height="50">](https://github.com/vperic)<a name="note_4"></a>|@vperic commented|
|-|-|

(In [35181]) Create branch remove-timehelpers-5820

**Comment by twisted-trac** [comment]
|[<img alt="vperic's avatar" src="https://avatars.githubusercontent.com/u/688281?s=50" width="50" height="50">](https://github.com/vperic)<a name="note_5"></a>|@vperic commented|
|-|-|

[Build results.](http://buildbot.twistedmatrix.com/boxes-supported?branch=/branches/remove-timehelpers-5820)

**Comment by twisted-trac** [comment]
|[<img alt="vperic's avatar" src="https://avatars.githubusercontent.com/u/688281?s=50" width="50" height="50">](https://github.com/vperic)<a name="note_6"></a>|@vperic commented|
|-|-|

Right, the usual Windows failures and something on FreeBSD (but I can't see how it can be related). I'm going to go ahead and merge.

**Comment by twisted-trac** [comment]
|[<img alt="vperic's avatar" src="https://avatars.githubusercontent.com/u/688281?s=50" width="50" height="50">](https://github.com/vperic)<a name="note_7"></a>|@vperic set status to `closed`|
|-|-|

(In [35183]) Merge remove-timehelpers-5820: Remove twisted.test.time_helpers

Author: vperic
Reviewer: thijs
Fixes: [#5820](https://github.com/twisted/twisted/issues/5820)

The module is deprecated since Twisted 10.0 and has a replacement
available (twisted.internet.task.Clock), so remove it and its' tests.
